File: client/src/app/_dtos/hr/candidateAssessedDto.ts
export interface ICandidateAssessedDto
{
     id: number;
     checked: boolean;
     orderItemId: number;
     orderId: number;
     professionName: string;
     applicationNo: number;
     customerName: string;
     requireInternalReview: boolean;
     candidateId: number;
     candidateName: string;
     assessedByUsername: string;
     assessResult: string;
     assessedOn: Date;
     checklistHRId: number;
     checklistedByName: string;
     checklistedOn: Date;
     categoryRef: string;
     assessedByName: string;
}

export class CandidateAssessedDto implements ICandidateAssessedDto
{
     id= 0;
     checked= false;
     orderItemId= 0;
     orderId=0;
     professionName= '';
     applicationNo= 0;
     customerName= '';
     requireInternalReview= false;
     candidateId= 0;
     candidateName= '';
     assessedByUsername= '';
     checklistHRId= 0;
     assessResult= '';
     assessedOn: Date = new Date();

     checklistedByName = '';
     checklistedOn = new Date();
     categoryRef = '';
     assessedByName = '';
}

File: api/DTOs/HR/CandidateAssessedDto.cs
namespace api.DTOs.HR
{
    public class CandidateAssessedDto
    {
        public int Id { get; set; }
        public bool Checked { get; set; } = false;
        public int OrderItemId { get; set; }
        public int OrderId { get; set; }
        public string ProfessionName { get; set; }
        public int ApplicationNo { get; set; }
        public string CustomerName { get; set; }
        public int CandidateId {get; set;}
        public string CandidateName { get; set; }
        public DateOnly AssessedOn { get; set; }
        public string AssessedByUsername {get; set;}
        public string RequireInternalReview {get; set;}
        public int ChecklistHRId {get; set;}
        public string AssessResult { get; set; } = "Not Assessed";
    }
}